### DISCUSSION TITLE
Bind to specific IP

### DISCUSSION THREAD

**Discussion by eugenewolf** [jhuckaby/Cronicle#576] [Q&A]
I have a server with severalIP. How can I force it to work on a specific IP?

**Comment by jhuckaby** [comment]
In your configuration file, add this into your `WebServer` object:

https://github.com/jhuckaby/pixl-server-web#http_bind_address

**Reply by eugenewolf** [reply]
Thanx I'm interested in the installation IP for the master server for websocket
